Plugins/IntelliJ/build.gradle.kts

```diff
@@ -60,6 +60,6 @@ tasks {
     runIde {
         // Absolute path to installed target 3.5 Android Studio to use as
         // IDE Development Instance (the "Contents" directory is macOS specific):
-        ideDir.set(file("/Applications/Android Studio.app/Contents"))
+        ideDir.set(file("/Applications/Android Studio Ladybug | 2024.2.1 Canary 9.app/Contents"))
```

Comment:
Is this from having multiple Android Studio versions installed or have they changed the default install naming?

Reply:
This is an oversight on my part. Didn't mean to check out that change. The default name for Android Studio hasn't changed, I was just testing this with an install of Canary 9 that I had installed side-by-side with my normal Android Studio.

I'll revert this change